tfjs-backend-webgpu/src/device_limits.ts

```diff
@@ -15,12 +15,11 @@
  * =============================================================================
  */
 
-export const MAX_COMPUTE_WORKGROUP_INVOCATIONS = 256;
+export let maxComputeWorkgroupsPerDimension = 256;
```

Comment:
I think the default `maxComputeWorkgroupsPerDimension` is 65535 according to the spec https://www.w3.org/TR/webgpu/#limits. You can link this address above the line as the annotation.

Reply:
Please also expose `maxComputeWorkgroupStorageSize` since we want to use it to do more optimizations.

Reply:
Done.